types/api/blocks.d.ts

```diff
@@ -26,8 +26,9 @@ export interface Blocks {
 
   /**
    * Removes current Block
+   * @param {number} index
    */
-  delete(): void;
+  delete(index: number): void;
```

Comment:
maybe it would better to make this parameter optional?

Reply:
So that it will work with currentBlockValue? Ok, I'll change it.